invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/creatibutors.html

```diff
@@ -39,7 +39,7 @@
 
 
 {% macro show_creatibutors(creatibutors, show_affiliations=False, type="creators", show_role=False) %}
-  {% for creatibutor in creatibutors %}
+  {% for creatibutor in creatibutors if creatibutor.person_or_org and creatibutor.person_or_org.name %}
```

Comment:
You can edit a contributor while on community review, remove the name while keeping the required role, and save. the reviewer will get the error.